dataset/dictionary/y-events.json5

```
@@ -356,52 +391,60 @@
   {
     en: "Hues of the Violet Garden",
     ja: "華やぐ紫苑の庭",
+    zhCN: "堇庭华彩",
     pronunciationJa: "はなやぐしおんのにわ",
     notes: "v2.6 期間限定イベント",
     tags: [ "event" ],
   },
   {
-    en: "Magnificent Irodori Festival",
+    en: "Maganificent Irodori Festival",
```

According to the official PV, "Magnificent" seems correct.
https://youtu.be/LqCwQicfMuc?t=165

```suggestion
    en: "Magnificent Irodori Festival",
```

---

You are right. I checked the dictionary and it seems that I have been spelling this word wrong for a long time LOL.